metaworld/envs/mujoco/sawyer_xyz/sawyer_assembly_peg.py

```diff
@@ -6,24 +6,25 @@
 
 
 class SawyerNutAssemblyEnv(SawyerXYZEnv):
+    goal_low = (-0.1, 0.75, 0.1)
+    goal_high = (0.1, 0.85, 0.1)
+    goal_space = Box(np.array(goal_low), np.array(goal_high))
```

Comment:
We should perhaps hide these fields? I'm not too concerned either way.

Reply:
it doesn't hurt to make them double-underscore and DO_NOT_USE

`__GOAL_LOW_DO_NOT_USE`

We need to remember that our audience is not expert Python programmers and that they are accustomed to accessing the private state of poorly-designed code to get their work done.